docs/docusaurus/docs/core/configure_project_settings/configure_metadata_stores/configure_metadata_stores.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Configure project Stores
+description: Configure the location of stored metadata and Validation Results for a File Data Context.
+hide_feedback_survey: false
+hide_title: false
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import PrereqPythonInstalled from '../../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../../_core_components/prerequisites/_gx_installation.md'
+import PrereqFileDataContext from '../../_core_components/prerequisites/_file_data_context.md'
+
+
+Stores are used by GX to store and retreive information ranging from project metadata such as Expectation Suite configurations to the Validation Results generated when Checkpoints are run.
+
+Ephemeral Data Contexts store this information in memory, while GX Cloud Data Contexts store this information online.  File Data Contexts, however, store this information in files thata can be copied or shared between other File Data Contexts.
+
+By default, Store files are created in folders within the `base_folder` of the File Data Context.  However, you can update your Data Context to specify where these Stores should reside, or to indicate existing Stores to load when the Data Context is initialized.
+
+### Prerequisites:
+
+- <PrereqPythonInstalled/>.
+- <PrereqGxInstalled/>.
+- <PrereqFileDataContext/>.
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Load a File Data Context.
+
+   Because Store configurations are loaded when a Data Context is initialized, an Ephemeral Data Context will always initialize with default in-memory Stores and any changes to them will not persist when a new Ephemeral Data Context is initialized. 
+
+   GX Cloud accounts manage Stores for you online, and do not support custom Store configurations.
+
+   Therefore, only File Data Contexts can have customized Store configurations.  This procedure assumes you have a File Data Context loaded as the variable `context`:
+
+   ```python title="Python"
+   import great_expectations as gx
+   
+   context = gx.get_context(mode="file")
+   ```
+
+2. Determine the Store to update.
+
+   GX utilizes 5 Stores for different types of data and metadata.  These Stores are the Expectations Store, Validation Definitions Store, Checkpoint Store, Validation Results Store, and the Suite Parameter Store.  All Stores can be accessed by passing a corresponding key to a Data Context's `variables.config.stores` attribute.  The following code shows how to print the configuration for each of these stores:
```

Comment:
If you don't use `.config` you will see the resolved store name after variable substitution. If you use `.config` you will see the raw name without substitution. Do we want to make that distinction here?